LordUche/sandbox
# #162735861 Initial setup

#### What does this PR do?
Sets up the app

#### What tasks need to be completed
- Install and configure Babel
- Enable linting with Eslint (AirBnB style guide)
- Write one passing test with Mocha and Chai
- Create MVC directory structure
- Create Express server
- Define base URL, catch-all and error handler routes
- Host API on Heroku
- Continuous Integration with Travis CI and test coverage with Coveralls
- Install and use Helmet for security

#### How should this be manually tested?
- Visit https://ireporter-sandbox.herokuapp.com/ and ensure that all specified routes are covered
- Ensure that all tests pass on Travis CI

#### Relevant board stories
#162735861 Initial setup
